packages/core/src/blocks/ListItemBlockContent/CheckListItemBlockContent/CheckListItemBlockContent.ts

```diff
@@ -193,35 +201,23 @@ const checkListItemBlockContent = createStronglyTypedTiptapNode({
       if (node.attrs.checked) {
         checkbox.setAttribute("checked", "");
       }
-      checkbox.disabled = !editor.isEditable;
 
       const changeHandler = () => {
-        if (typeof getPos === "boolean") {
-          return;
+        if (!editor.isEditable) {
+          checkbox.checked = !checkbox.checked;
```

Comment:
Is this necessary or can we also use preventDefault or return false to prevent the change?

if it's necessary, please comment in code why we need this (it's a typical piece of code that might raise questions / confuse ourselves later!)